project page.html

```diff
@@ -0,0 +1,46 @@
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="/style.css">
+</head>
+<body>
+    <main id="main">
+        <h2 id="title">Dr. Norman Borlaug</h2>
+        <p>The man who saved a billion lives</p>
+            
+        <br />
+        <figure id="img-div">
+            <img id="image" src="https://c2.staticflickr.com/4/3689/10613180113_fdf7bcd316_b.jpg" alt="Dr. Bowman and friends">
+            <figcaption id="img-caption">Dr. Norman Borlaug, third from the left, trains biologists in Mexico on how to increase wheat yields - part of his life-long war on hunger.</figcaption>
+        </figure>
+       <div id="tribute-info">
+            <h3>Here's a time line of Dr. Borlaug's life:</h3>
+            <ul id="ordered-list">
+                <li><strong>1914</strong>  - Born in Cresco, Iowa</li>
+                <li><strong>1935</strong> - Has to stop school and save up more money. Works in the Civilian Conservation Corps, helping starving Americans. "I saw how food changed them", he said. "All of this left scars on me."</li>
+                <li><strong>1937</strong> - Finishes university and takes a job in the US Forestry Service</li>
+                <li><strong>1938</strong> - Marries wife of 69 years Margret Gibson. Gets laid off due to budget cuts. Inspired by Elvin Charles Stakman, he returns to school study under Stakman, who teaches him about breeding pest-resistent plants.</li>
+                <li><strong>1942</strong> - Receives a Ph.D. in Genetics and Plant Pathology</li>
+                <li><strong>1970</strong> - receives the Nobel Peace Prize</li>
+                <li><strong>1914</strong>  - Born in Cresco, Iowa</li>
+                <li><strong>1935</strong> - Has to stop school and save up more money. Works in the Civilian Conservation Corps, helping starving Americans. "I saw how food changed them", he said. "All of this left scars on me."</li>
+                <li><strong>1937</strong> - Finishes university and takes a job in the US Forestry Service</li>
+                <li><strong>1938</strong> - Marries wife of 69 years Margret Gibson. Gets laid off due to budget cuts. Inspired by Elvin Charles Stakman, he returns to school study under Stakman, who teaches him about breeding pest-resistent plants.</li>
+                <li><strong>1942</strong> - Receives a Ph.D. in Genetics and Plant Pathology</li>
+                <li><strong>1970</strong> - receives the Nobel Peace Prize</li>
+            </ul>
+        </div>
+        <div id="tribute">
+          <p>If you have time, you should read more about this incredible human being on his <a id="tribute-link" href="https://en.wikipedia.org/wiki/Norman_Borlaug
+        " rel="noopener" target="_blank">Wikipedia Link</a> </p> 
+        </div>
+        
+    </main>
+
+    <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
+    <script  src="js/index.js"></script>
+</body>
+</html>
```

Comment:
As a good practice, I suggest you always add a new empty line at the end of your codes. This is because it's a good practice and adopted as many company's standards. 😁

Reply:
This is understood and changes would be done. Thanks